RCAIDE/Framework/Missions/Initialize/energy.py

```diff
@@ -0,0 +1,55 @@
+# RCAIDE/Framework/Missions/Initialization/energy.py
+# (c) Copyright 2024 Aerospace Research Community LLC
+# Created: Aug 2024, RCAIDE Team
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Imports
+# ----------------------------------------------------------------------------------------------------------------------
+
+# package imports
+import numpy as np
+
+# RCAIDE Imports
+import RCAIDE.Framework as rcf
+from RCAIDE.Framework.Missions.Conditions import Conditions
+from RCAIDE.Framework.Missions.Conditions.Energy import EnergyLineConditions as Line
+from RCAIDE.Framework.Missions.Conditions.Energy import EnergyStoreConditions as Store
+from RCAIDE.Framework.Missions.Conditions.Energy import EnergyConverterConditions as Converter
+
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Initialize Energy
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+def initialize_energy(state: "rcf.State",
+                      system: "rcf.Aircraft",
+                      settings: "rcf.Settings",
+                      ):
+
+    for l_idx, line in enumerate(system.energy.lines.subcomponents):
+        state.energy.lines[l_idx] = Line()
+        for p_idx, propulsor in enumerate(line.propulsors.subcomponents):
+            state.energy.lines[l_idx].propulsors[p_idx] = Converter()
```

Comment:
This nomenclature is a bit inconsistent because a propulsor is being set with converter conditions